### ISSUE TITLE
App does not work when adblock enabled.

### ISSUE THREAD

**Issue by potherca** [mstefaniuk/graphviz.it#8]
When trying to use the app, nothing happens. On further inspection, the following error is present in the console:

    analytics.js:2 Uncaught TypeError: Cannot set property 'l' of undefined

This is caused by an ad-blocker/privacy plugin that stops the Google Analytics JS from loading.

To prevent this error from breaking the application a check that `ga` actually exists should be added.

**Comment by mstefaniuk** [comment]
@potherca It's a little bit strange. For webpage bootstrap `require.js` is used with special fallback for Google analytics:
```javascript
    ga: [
      '//www.google-analytics.com/analytics',
      'gamock'
    ]
```
so `gamock.js` should be loaded instead of original analytics. It is an empty function and as far I've tested it works correctly when original URL is wrong (in Firefox).

Could you provide more details about the privacy plugin you used? And maybe names and versions of used browsers?

**Comment by potherca** [comment]
The details, as requested, my apologies for not including these in the first place :smile: 

Browser: Google Chrome, Version 47.0.2526.106 (64-bit)
Plugin: [Ghostery](https://chrome.google.com/webstore/detail/ghostery/mlomiejdfkolichcflejclcbmpeaniij)

When Ghostery is disabled/paused all works as expected. The workaround for me was to "alway allow Google Analytics on graphviz.it"

<img width="663" alt="screen shot 2015-12-27 at 22 21 43" src="https://cloud.githubusercontent.com/assets/195757/12012128/59ff7190-ace8-11e5-80a3-408157d51d58.png">

**Comment by potherca** [comment]
I can confirm the fix in 121f7c8267f6a959c03ddd1521c44f4525bfa856 works when Ghostery is enabled.
Thank you very much! :smiley::+1:
